shallowblue_rllib.py

```diff
@@ -98,49 +90,45 @@ def step(self, action):
         Take an action in the environment and return the results.
 
         Args
-            action: <int> index of the action to take
+            action: 4 tuple corresponding to continous value of each action
 
         Returns
             observation: <np.array> flattened array of obseravtion
             reward: <int> reward from taking action
             done: <bool> indicates terminal state
             info: <dict> dictionary of extra information
         """
+        # allow_break_action = self.obs[1, int(self.obs_size/2)-1, int(self.obs_size/2)] == 1
 
         # Get Action
-        command = self.action_dict[action]
-        allow_break_action = self.obs[1, int(self.obs_size/2)-1, int(self.obs_size/2)] == 1
-
-        if command == 'attack 1' and allow_break_action:
-            self.agent_host.sendCommand(command)
-            time.sleep(.1)
-            self.episode_step += 1
-        elif command == 'attack 1':
-            pass
-        elif command == 'jump 1' and self.air_level > (self.full_air * .3): 
-            pass
-        elif command == 'jump 0' and self.air_level < (self.full_air * .3):
-            pass
+        if self.allow_break_action and action[3] > 0:
+            self.agent_host.sendCommand('move 0')
+            self.agent_host.sendCommand('turn 0')
+            self.agent_host.sendCommand('jump 0')
+            self.agent_host.sendCommand('attack 1')
+            time.sleep(1.5)
```

Comment:
i made this time.sleep longer since i think the agent takes longer to mine underwater